base/audit_types.go

```diff
@@ -133,7 +134,7 @@ func (f AuditFields) expandMandatoryFieldGroups(groups []fieldGroup) {
 
 func (i AuditID) MustValidateFields(f AuditFields) {
 	if err := i.ValidateFields(f); err != nil {
-		panic(fmt.Errorf("audit event %s(%s) invalid:\n%v", AuditEvents[i].Name, i, err))
+		panic(fmt.Errorf("audit event %q (%s) invalid:\n%v", i, AuditEvents[i].Name, err))
```

Comment:
I changed this just to align with the existing assertion messages in `TestAuditLoggingFields`